### DISCUSSION TITLE
Deploy in a sub-path

### DISCUSSION THREAD

**Discussion by rdpascua** [pocketbase/pocketbase#1802] [Q&A]
I tried copying the said configuration here
https://github.com/pocketbase/pocketbase/issues/223

I want to be able to view the pocketbase admin `https://domain.com/**admin**`

```
 location /admin {
        proxy_set_header Connection '';
        proxy_http_version 1.1;
        proxy_read_timeout 180s;

        proxy_set_header Host $host;
        proxy_set_header X-Real-IP $remote_addr;
        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
        proxy_set_header X-Forwarded-Proto $scheme;

        rewrite /admin/(.*) /$1  break;
        proxy_redirect     off;
        proxy_pass http://localhost:8090;
    }
```

This configuration works but the pb is still calling `https://domain.com/api/admins`, anyone knows how to fix this issue?

**Comment by ganigeorgiev** [comment]
Could you clarify what do you mean by _"This configuration works but the pb is still calling https://domain.com/api/admins"_?


The expected working endpoints should be:

- https://domain.com/admin/_/
- https://domain.com/admin/api

right?

**Reply by ganigeorgiev** [reply]
I've just tested it and it should work but there are 2 caveats that will be fixed in the near future:

1. The admin endpoint must be with ending slash because the api url in the UI currently is defined as the relative `../`. As a workaround you can define a redirect location:
    ```nginx
    location = /admin/_ {
      return 302 $uri/;
    }
    ```

2. If there is no admin, the user will be always redirected server-side to `/_/?installer`.
I've added it in my todo and will try to update the handling in the next v0.13 or v0.14 release.

**Reply by rdpascua** [reply]
Yes this is exactly what I'm having on a fresh install, it gets redirected to `/_/?installer`.

For some reason it's calling the the `/api/admins` where it should be calling `/**admin**/api/admins`

**Reply by ganigeorgiev** [reply]
_(Just for info in case someone stumble on this)_

The subpath handling was updated in the [latest v0.13.0 release](https://github.com/pocketbase/pocketbase/releases/tag/v0.13.0) and the extra `location` redirects should no longer be necessary. Here is an example of a minimal subpath nginx configuration:

```nginx
server {
    listen 80;
    server_name example.com;
    client_max_body_size 10M;

    location /pb {
        # check http://nginx.org/en/docs/http/ngx_http_upstream_module.html#keepalive
        proxy_set_header Connection '';
        proxy_http_version 1.1;
        proxy_read_timeout 360s;

        proxy_set_header Host $host;
        proxy_set_header X-Real-IP $remote_addr;
        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
        proxy_set_header X-Forwarded-Proto $scheme;

        rewrite /pb/(.*) /$1  break;

        proxy_pass http://127.0.0.1:8090;
    }
}
```
